### ISSUE TITLE
version() test failure when building on ArchLinux

### ISSUE THREAD

**Issue by hubutui** [xflr6/graphviz#87]
Hi, the test fails on the 0.13.1 release, here the error message:
```
==================================== ERRORS ====================================
____________ ERROR at setup of test_pipe_invalid_data[quiet=False] _____________
Error evaluating 'xfail' expression
    version() == (2, 36, 0)
RuntimeError: cannot parse ['dot', '-V'] output: 'dot - graphviz version 2.43.20190912.0211 (20190912.0211)\n'

------------------------------ Captured log setup ------------------------------
DEBUG    graphviz.backend:backend.py:152 run ['dot', '-V']
_____________ ERROR at setup of test_pipe_invalid_data[quiet=True] _____________
Error evaluating 'xfail' expression
    version() == (2, 36, 0)
RuntimeError: cannot parse ['dot', '-V'] output: 'dot - graphviz version 2.43.20190912.0211 (20190912.0211)\n'

------------------------------ Captured log setup ------------------------------
DEBUG    graphviz.backend:backend.py:152 run ['dot', '-V']
=================================== FAILURES ===================================
_________________________________ test_version _________________________________

capsys = <_pytest.capture.CaptureFixture object at 0x7f9d8edb6690>

    @pytest.exe
    def test_version(capsys):
>       result = version()

tests/test_backend.py:219: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

    def version():
        """Return the version number tuple from the ``stderr`` output of ``dot -V``.
    
        Returns:
            Two or three ``int`` version ``tuple``.
        Raises:
            graphviz.ExecutableNotFound: If the Graphviz executable is not found.
            subprocess.CalledProcessError: If the exit status is non-zero.
            RuntimmeError: If the output cannot be parsed into a version number.
        """
        cmd = ['dot', '-V']
        out, _ = run(cmd, check=True,
                     stdout=subprocess.PIPE,
                     stderr=subprocess.STDOUT)
    
        info = out.decode('ascii')
        ma = re.search(r'graphviz version (\d+\.\d+(?:\.\d+)?) ', info)
        if ma is None:
>           raise RuntimeError('cannot parse %r output: %r' % (cmd, info))
E           RuntimeError: cannot parse ['dot', '-V'] output: 'dot - graphviz version 2.43.20190912.0211 (20190912.0211)\n'

graphviz/backend.py:251: RuntimeError
------------------------------ Captured log call -------------------------------
DEBUG    graphviz.backend:backend.py:152 run ['dot', '-V']
=============================== warnings summary ===============================
tests/test_lang.py:14
  /build/python-graphviz/src/graphviz-0.13.1/tests/test_lang.py:14: DeprecationWarning: invalid escape sequence \G
    ('\G \E \T \H \L', '"\\G \\E \\T \\H \\L"'),  # noqa: W605

tests/test_lang.py:16
  /build/python-graphviz/src/graphviz-0.13.1/tests/test_lang.py:16: DeprecationWarning: invalid escape sequence \l
    ('\l', '"\\l"'),  # noqa: W605
```
I'm building `python-graphviz` in a clean chroot ArchLinux. The full build log is here: [log.txt](https://github.com/xflr6/graphviz/files/3819606/log.txt)

**Comment by xflr6** [comment]
Thanks. Can you re-try with f137dc45676404c9bc6156667ffc9a3b9c6ed5f3?

**Comment by hubutui** [comment]
OK, I can confirm that this commit fix the issue. Would you like to bump the version? If not, I'll create a patch for building this package for ArchLinux.

**Comment by xflr6** [comment]
I don't think a patch is needed. Fixed in [0.13.2 release](https://pypi.org/project/graphviz/0.13.2/#files) (d61e77fdfebb8a0cd8cd9e74f55e9ddda0c7410d). Thanks for the report.
